main/src/com/google/refine/exporters/CsvExporter.java

```diff
@@ -74,12 +72,12 @@ public CsvExporter(char separator) {
     @Override
     public void export(Project project, Properties params, Engine engine, final Writer writer)
             throws IOException {
-        
+
         String optionsString = (params == null) ? null : params.getProperty("options");
-        Configuration options = new Configuration();
+        com.google.refine.exporters.CsvExporter.Configuration options = new com.google.refine.exporters.CsvExporter.Configuration();
```

Comment:
If this import conflicts with another one, then I would rather import our own class, and keep the fully qualified name for the other class (the external one).